src/RouterSettings/Ports.js

```diff
@@ -44,14 +44,33 @@ const Ports = () => {
 
   useInterval(getInterfaces, 5000);
 
-  let setInterfaceMode = (iface, mode) => {
-    setSelected(iface);
-    if (mode === "Wan") {
+  // note (TODO) iface has been set on backend to accept a Vec<string> of interfaces, and mode a Vec<InterfaceMode>
+  function setInterfaceMode() {
+    //   //TODO change this back to before- but clear selected. this fn will now be called on save button
+    //   //setSelected(selected.push(iface));
+    //   //if (new_mode === "Wan") {
```

Comment:
not sure if these todo's need to be resolved

Reply:
Whoops, forgot to delete the last of the TODOs. fixed